### DISCUSSION TITLE
Syntax highlighting doesn't work

### DISCUSSION THREAD

**Discussion by green-study** [facebook/docusaurus#9689] [Q&A] [answered]
I try to use code block on the docusaurus.
But, It doesn't work.
I added additionalLanguages: ['python'] already.

.md file code
![image](https://github.com/facebook/docusaurus/assets/70996958/5ab8bfaf-dd8e-4cb8-a2d2-5c3adf33b974)

website
![image](https://github.com/facebook/docusaurus/assets/70996958/b1875cd6-80d8-4806-bce4-ed94eb6addee)

html
![image](https://github.com/facebook/docusaurus/assets/70996958/01ade564-1e6a-4a6c-b1fe-315884493fc4)

environment
```
{
  "name": "doc-v-3",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "docusaurus": "docusaurus",
    "start": "docusaurus start",
    "build": "docusaurus build",
    "swizzle": "docusaurus swizzle",
    "deploy": "docusaurus deploy",
    "clear": "docusaurus clear",
    "serve": "docusaurus serve",
    "write-translations": "docusaurus write-translations",
    "write-heading-ids": "docusaurus write-heading-ids"
  },
  "dependencies": {
    "@docusaurus/core": "^3.0.1",
    "@docusaurus/plugin-google-gtag": "^3.0.1",
    "@docusaurus/plugin-pwa": "^3.0.1",
    "@docusaurus/preset-classic": "^3.0.1",
    "@docusaurus/theme-common": "^3.0.1",
    "@mdx-js/react": "^1.6.22",
    "classnames": "^2.3.2",
    "clsx": "^1.2.1",
    "docusaurus-lunr-search": "^2.3.2",
    "prism-react-renderer": "^1.3.5",
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
  },
  "devDependencies": {
    "@docusaurus/module-type-aliases": "^3.0.1"
  },
  "browserslist": {
    "production": [
      ">0.5%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  },
  "engines": {
    "node": ">=16.14"
  }
}
```

**Comment by slorber** [comment]
please provide a runnable repro using docusaurus.new

**Comment by Josh-Cena** [comment, marked as answer]
You are still using `prism-react-renderer` v1. Please upgrade to the latest version (`prism-react-renderer@2.3.1`) which includes python out-of-the-box. Also you should upgrade `@mdx-js/react` to v3 too; please follow https://docusaurus.io/blog/releases/3.0#breaking-changes to upgrade all dependencies

**Comment by green-study** [comment]
It looks like the version problem.
There's a third-party and dependency problem, too.
I'll try to solve it.
Thank you for answering.
